recipe/deploy/update_code.php

```diff
@@ -27,8 +27,15 @@
     $repository = get('repository');
     $target = get('target');
 
-    if (get('auto_ssh_keygen')) {
-        $url = parse_url($repository);
+    $skipKeygen = false;
+    $url = parse_url($repository);
+
+    // Skip keygen, if URL is using http/https as scheme
+    if (isset($url['scheme']) && in_array($url['scheme'], ['http', 'https'])) {
+        $skipKeygen = false;
+    }
+
+    if (get('auto_ssh_keygen') && $skipKeygen) {
```

Comment:
I think it should be `!$skipKeygen`.

Reply:
Sorry, I have changed the logic and forgot to update the rest...